lib/ain-rs-exports/src/evm.rs

```diff
@@ -456,11 +456,17 @@ pub fn evm_unsafe_try_construct_block_in_q(
                 total_priority_fees,
                 block_number,
             }) => {
-                let Ok(total_burnt_fees) = u64::try_from(WeiAmount(total_burnt_fees).to_satoshi()) else {
+                let Ok(total_burnt_fees) = u64::try_from(WeiAmount(total_burnt_fees).to_satoshi())
+                else {
                     return cross_boundary_error_return(result, "total burnt fees value overflow");
                 };
-                let Ok(total_priority_fees) = u64::try_from(WeiAmount(total_priority_fees).to_satoshi()) else {
```

Comment:
Same for this? Also the formatting is inconsistent with the above for the ease of reading (although clippy check passes)

Reply:
hmm.. is your fmt good which didn't trigger this alert??
lemme drop this.. just with `--no-verify`